internal/pkg/agent/vault/seed.go

```diff
@@ -16,14 +17,23 @@ import (
 )
 
 const (
+	// seedFile is len(aesgcm.AES256) and contains only the random seed
+	// A default salt size of 8 is used with this seed file
 	seedFile = ".seed"
+	// seedFileV2 is len(aesgcm.AES256+4) and contains the random seed followed by a non-zero salt size (little endian uint32)
+	seedFileV2 = ".seedV2"
```

Comment:
Currently, we have hard-coded values for iterations, and keyLen. Should we add these to the seed file to allow for changes?